web/libs/datamanager/src/components/Common/Range/Range.scss

```
@@ -0,0 +1,132 @@
+.range {
+  display: inline-flex;
+  align-items: center;
+  user-select: none;
+
+  &_align {
+    &_horizontal {
+      min-width: 120px;
+      height: 28px;
+      flex-direction: row;
+    }
+
+    &_vertical {
+      min-height: 120px;
+      width: 28px;
+      flex-direction: column;
+    }
+
+    .range_with-icon &_horizontal {
+      padding: 0 5px;
+    }
+
+    .range_with-icon &_vertical {
+      padding: 5px 0;
+    }
+  }
+
+  &__body {
+    position: relative;
+
+    .range_align_horizontal & {
+      height: 4px;
+      width: 100%;
+    }
+
+    .range_align_vertical & {
+      height: 100%;
+      width: 4px;
+    }
+
+    .range_align_horizontal &_with-icon {
+      margin: 0 5px;
+    }
+
+    .range_align_vertical &_with-icon {
+      margin: 5px 0;
+    }
+  }
+
+  &__line {
+    background: linear-gradient(0deg, rgba(0, 0, 0, 0.05), rgba(0, 0, 0, 0.05)), #FFFFFF;
+    box-shadow: inset 0px 1px 0px rgba(0, 0, 0, 0.05), inset 0px 0px 0px 1px rgba(0, 0, 0, 0.05);
+    border-radius: 8px;
+
+    .range_align_horizontal & {
+      height: 4px;
+      width: 100%;
+    }
+
+    .range_align_vertical & {
+      width: 4px;
+      height: 100%;
+    }
+  }
+
+  &__indicator {
+    border-radius: 8px;
+    position: absolute;
+    background-color: rgba(var(--accent_color), 0.5);
+
+    .range_align_horizontal & {
+      top: 0;
+      height: 4px;
+    }
+
+    .range_align_vertical & {
+      left: 0;
+      width: 4px;
+    }
+  }
+
+  &__range-handle {
+    width: 0;
+    height: 0;
+    position: absolute;
+
+    &::before {
+      content: '';
+      position: absolute;
+      background: var(--grape_500);
+      box-shadow: 0px 5px 10px rgba(0, 153, 255, 0.3), inset 0px -1px 0px rgba(0, 0, 0, 0.1);
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
      box-shadow: 0px 5px 10px rgba(0, 153, 255, 0.3), inset 0px -1px 0px var(--black_10);
```